content/pages/regler.mdx

```diff
@@ -1,10 +1,17 @@
 ---
 title: Regler
-description: fawekgnaejkgnewkøasg
+description: Reglene til Queer Hangout's arrangementer
 slug: /regler
-menu: 2
+menu: 3
 ---
 
 # Regler
+  
+![The safe and inclusive space guidelines, made by Skeiv Verden. One, inclusion, everyone should be included, seen and weelcome. Two, pronouns, respect the pronoun people use. Ask if you don't know. Three, tone of communication, use respectful language, avoid jokes which can easily be misunderstood. Four, touching, ask before making physical contact. People have different needs for personal space. Five, flirting, be mindful of not crossing other people's boundaries, flirting can make some people uncomfortable. Six, privacy, everyone has a different story and can decide what they want to share about themselves. It's okay to say no. Seven, confidentiality, respect anonymity. Do not share personal information of others. Eight, cameras, photos should always be taken with consent. Nine, open mind, misunderstandings happen, take care of each other. Relax, everyone makes mistakes! Accept and or give feedback in a constructive way. And be kind, always!](../images/safespage.jpg "Brukt med tillatelse fra Skeiv Verden") <figcaption>Bilde brukte med tillatelse fra <a href="https://skeivverden.no">Skeiv Verden</a></figcaption>  
```

Comment:
Her kan du fjerne figcaption, for den blir overflødig når tittel også brukes som caption :)